percona-server/Dockerfile

```diff
@@ -1,51 +1,56 @@
             -c "Default Application User" mysql
 
 ENV PERCONA_MAJOR 5.7
-ENV PERCONA_VERSION 5.7.17-11-1.jessie
-
+ENV PERCONA_VERSION 5.7.17-13-1.jessie
 
 # the "/var/lib/mysql" stuff here is because the mysql-server postinst doesn't have an explicit way to disable the mysql_install_db codepath besides having a database already "configured" (ie, stuff in /var/lib/mysql/mysql)
 # also, we set debconf keys to make APT a little quieter
 RUN { \
-		echo percona-server-server-$PERCONA_MAJOR percona-server-server/root_password password 'unused'; \
-		echo percona-server-server-$PERCONA_MAJOR percona-server-server/root_password_again password 'unused'; \
-	} | debconf-set-selections \
-	&& apt-get update \
-	&& apt-get install -y --force-yes \
-		percona-server-server-$PERCONA_MAJOR=$PERCONA_VERSION \
-	&& rm -rf /var/lib/apt/lists/* \
+                echo percona-server-server-$PERCONA_MAJOR percona-server-server/root_password password 'unused'; \
+                echo percona-server-server-$PERCONA_MAJOR percona-server-server/root_password_again password 'unused'; \
+        } | debconf-set-selections \
+        && apt-get update \
+        && apt-get install --force-yes -y \
+                percona-server-server-$PERCONA_MAJOR=$PERCONA_VERSION percona-server-tokudb-$PERCONA_MAJOR=$PERCONA_VERSION\
+        && rm -rf /var/lib/apt/lists/* \
         && mv /etc/mysql/percona-server.conf.d/mysqld.cnf /etc/mysql/my.cnf \
 # comment out any "user" entires in the MySQL config ("docker-entrypoint.sh" or "--user" will handle user switching)
-	&& sed -ri 's/^user\s/#&/' /etc/mysql/my.cnf \
+        && sed -ri 's/^user\s/#&/' /etc/mysql/my.cnf \
 # purge and re-create /var/lib/mysql with appropriate ownership
-	&& rm -rf /var/lib/mysql && mkdir -p /var/lib/mysql /var/run/mysqld \
-	&& chown -R 1001:0 /var/lib/mysql /var/run/mysqld \
+        && rm -rf /var/lib/mysql && mkdir -p /var/lib/mysql /var/run/mysqld \
+        && chown -R 1001:0 /var/lib/mysql /var/run/mysqld \
 # ensure that /var/run/mysqld (used for socket and lock files) is writable regardless of the UID our mysqld instance ends up having at runtime
-	&& chmod 777 /var/run/mysqld
+        && chmod 777 /var/run/mysqld
 
 # comment out a few problematic configuration values
 # don't reverse lookup hostnames, they are usually another container
 RUN sed -Ei 's/^(bind-address|log)/#&/' /etc/mysql/my.cnf \
-	&& echo 'skip-host-cache\nskip-name-resolve' | awk '{ print } $1 == "[mysqld]" && c == 0 { c = 1; system("cat") }' /etc/mysql/my.cnf > /tmp/my.cnf \
-	&& mv /tmp/my.cnf /etc/mysql/my.cnf
+        && echo 'skip-host-cache\nskip-name-resolve' | awk '{ print } $1 == "[mysqld]" && c == 0 { c = 1; system("cat") }' /etc/mysql/my.cnf > /tmp/my.cnf \
+        && mv /tmp/my.cnf /etc/mysql/my.cnf
 
 VOLUME ["/var/lib/mysql", "/var/log/mysql"]
 
 RUN sed -Ei '/log-error/s/^/#/g' -i /etc/mysql/my.cnf
 
+RUN echo "\n[mysqld_safe]\nthp-setting=never" >> /etc/mysql/my.cnf
```

Comment:
ps_tokudb_admin does this because our packages work with mysqld_safe on most distributions, since docker container cannot modify THP on the host anyway I think this part is not needed here. Simply if THP is enabled on the host tokudb won't be loaded, if it is disabled it will be loaded but it needs to be part of the documentation that THP needs to be disabled on the host for tokudb to work.